qgreenland/util/cleanup.py

```diff
@@ -172,13 +171,6 @@ def cleanup_cli(**kwargs):  # noqa: C901
             f'rm -rf {TaskType.FINAL.value}/*',
             dry_run=kwargs['dry_run'],
         )
-        # The triggerfile tells Luigi tasks to zip the compiled data. Can't do
-        # that if we just deleted it!
-        if os.path.isfile(ZIP_TRIGGERFILE):
```

Comment:
TODO: We should probably still do something to try and clean up the symbolic link that replaced the triggerfile. This would be left around if the zip process is interrupted.